lib/iframe-browser.js

```diff
@@ -90,6 +92,29 @@ define(function (require, exports, module) {
         return document.getElementById("bramble-iframe-browser");
     }
 
+    // Open a new window with the content of the iframe
+    // Only 1 window is allowed for the preview
+    function detachPreview() {
+        if(getBrowserIframe()) {
+            var iframe = getBrowserIframe();
+
+            if(!iframe) {
+                return;
+            }
+
+            var currentURL = getBrowserIframe().src;
+            if(!detachedWindow || detachedWindow.closed){
+                detachedWindow = window.open(currentURL, "Preview");
+            } else if(previousURL !== currentURL){
+                detachedWindow = window.open(currentURL, "Preview");
```

Comment:
What about my previous question re: closing the existing window before you do this?